<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ilxlrange</class>
 <widget class="QDialog" name="ilxlrange">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>222</width>
    <height>180</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>181</width>
    <height>100</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Inline byte #</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinboxInlineByte">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>10</y>
     <width>91</width>
     <height>22</height>
    </rect>
   </property>
   <property name="locale">
    <locale language="English" country="UnitedStates"/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::NoButtons</enum>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>300</number>
   </property>
   <property name="value">
    <number>9</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Xline byte #</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinboxXlineByte">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>40</y>
     <width>91</width>
     <height>22</height>
    </rect>
   </property>
   <property name="locale">
    <locale language="English" country="UnitedStates"/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::NoButtons</enum>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>300</number>
   </property>
   <property name="value">
    <number>21</number>
   </property>
  </widget>
  <widget class="QPushButton" name="btnClose">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>140</y>
     <width>71</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnStart">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>140</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>X coordinate byte #</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinboxYByte">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>100</y>
     <width>91</width>
     <height>22</height>
    </rect>
   </property>
   <property name="locale">
    <locale language="English" country="UnitedStates"/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::NoButtons</enum>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>300</number>
   </property>
   <property name="value">
    <number>85</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinboxXByte">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>70</y>
     <width>91</width>
     <height>22</height>
    </rect>
   </property>
   <property name="locale">
    <locale language="English" country="UnitedStates"/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::NoButtons</enum>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>300</number>
   </property>
   <property name="value">
    <number>81</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Y coordinate byte #</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btnClose</sender>
   <signal>clicked()</signal>
   <receiver>ilxlrange</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>135</x>
     <y>81</y>
    </hint>
    <hint type="destinationlabel">
     <x>90</x>
     <y>49</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
